sdk/core/core-asynciterator-polyfill/package.json

```diff
@@ -6,7 +6,7 @@
     "url": "https://github.com/Azure/azure-sdk-for-js"
   },
   "sdk-type": "client",
-  "version": "1.0.0-preview.1",
+  "version": "1.0.0-preview.2",
```

Comment:
We dont plan to make any changes to this package. Incrementing the version here and forcing all other packages to update their dependencies will only result us in reverting the change when we are ready to release the said packages

Reply:
We agreed offline to always increment package versions immediately after they ship, but only update dependencies when needed.  So it's correct to update this package version to `preview.2`, but you are correct all dependencies on this package should remain `preview.1` for now.